src/status_im2/subs/wallet_2/wallet.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im2.subs.wallet-2.wallet
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [utils.number]))
+
+(defn calculate-raw-balance
```

Comment:
should these helpers be private? i.e `defn-`

Reply:
Done